gradle-plugins/src/main/java/com/linkedin/pegasus/gradle/PegasusPlugin.java

```diff
@@ -913,6 +921,41 @@ protected void configureGeneratedSourcesAndJavadoc(Project project)
     }
   }
 
+  protected void configureExtensionSchemaValidation(Project project, SourceSet sourceSet)
```

Comment:
I'd rename this to configureExtensionSchemaValidationAndPublishTasks as this is not just validating (also publishing jars)

Reply:
Renamed this to configureExtensionSchemaValidationAndPublishTasks